runtime/moonriver/src/lib.rs

```diff
@@ -378,6 +386,7 @@ impl pallet_transaction_payment::Config for Runtime {
 	type WeightToFee = ConstantMultiplier<Balance, ConstU128<{ currency::WEIGHT_FEE }>>;
 	type LengthToFee = LengthToFee;
 	type FeeMultiplierUpdate = SlowAdjustingFeeUpdate<Runtime>;
+	type WeightInfo = ();
```

Comment:
We still need to added the weight info here.

Reply:
Addressed in https://github.com/moonbeam-foundation/moonbeam/pull/3224/commits/c1efa9bc7e992362b9c934cbb71ac05962c9ce57